js/timers.js

```diff
@@ -0,0 +1,70 @@
+// A collection of globally accessible timers
+
```

Comment:
I have concerns with this way of listing timers. first thing I can think of is there would be no way to stack two similar timers as they are only properties within a timers object. It makes more sense to me for timers to be an array and when a timer is created, it is added into the array stack. once a timer is finished, it is then disposed of. This way we can have multiple timers of the same type running at once, i.e. two attack up potion timers that stack and add to your  attack per each potion used.